### DISCUSSION TITLE
is there a documentation available, on how to connect to Snowflake through dbeaver

### DISCUSSION THREAD

**Discussion by ghost** [dbeaver/dbeaver#8958] [Q&A]
#### System information: 
- Operating system (distribution) and version
- DBeaver version

#### Connection specification:
- Database name and version

**Comment by LonwoLonwo** [comment]
You can connect to Snowflake through DBeaver Like to any other connection: https://github.com/dbeaver/dbeaver/wiki/Create-Connection

![2020-12-26 22_41_14-DBeaver 7 3 1](https://user-images.githubusercontent.com/45152336/103158134-bd9c4380-47cb-11eb-9e24-6ad9bafa3468.png)
![2020-12-26 22_41_28-DBeaver 7 3 1](https://user-images.githubusercontent.com/45152336/103158135-be34da00-47cb-11eb-878d-ddc38b312c9a.png)
